<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportSettingsDialog</class>
 <widget class="QDialog" name="ExportSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>569</width>
    <height>442</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Export Settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_7">
   <item row="0" column="0" rowspan="2">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>170</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string>Presets</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0">
       <widget class="QListWidget" name="listWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="showDropIndicator" stdset="0">
         <bool>false</bool>
        </property>
        <property name="defaultDropAction">
         <enum>Qt::IgnoreAction</enum>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QToolButton" name="toolButtonAddPreset">
          <property name="minimumSize">
           <size>
            <width>25</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>25</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Add and save current settings as preset</string>
          </property>
          <property name="text">
           <string>+</string>
          </property>
          <property name="icon">
           <iconset resource="ressources.qrc">
            <normaloff>:/RetinaIMG/RetinaIMG/plus.png</normaloff>:/RetinaIMG/RetinaIMG/plus.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButtonDeletePreset">
          <property name="minimumSize">
           <size>
            <width>25</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>25</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Delete selected preset from list</string>
          </property>
          <property name="text">
           <string>-</string>
          </property>
          <property name="icon">
           <iconset resource="ressources.qrc">
            <normaloff>:/RetinaIMG/RetinaIMG/minus.png</normaloff>:/RetinaIMG/RetinaIMG/minus.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>File Export</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="1">
         <widget class="QComboBox" name="comboBoxCodec">
          <item>
           <property name="text">
            <string>Apple ProRes 422 Proxy</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Apple ProRes 422 LT</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Apple ProRes 422</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Apple ProRes 422 HQ</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Apple ProRes 4444</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Uncompressed AVI</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>CinemaDNG Uncompressed</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>CinemaDNG Lossless</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>CinemaDNG Fast Pass</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>H.264</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>H.265</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>TIFF</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>PNG Sequence</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>JPEG2000</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Motion JPEG</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Huff YUV (FFVH)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>MLV</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>DNxHD</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>DNxHR</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>WAV (Audio Only)</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="comboBoxOption"/>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Codec</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="comboBoxDebayer">
          <item>
           <property name="text">
            <string>Force Bilinear</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Force AMaZE</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Force LMMSE</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Force IGV</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Receipt configuration</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="labelDebayer">
          <property name="text">
           <string>Debayer</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_Info">
          <property name="text">
           <string/>
          </property>
          <property name="scaledContents">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="3">
         <widget class="QSpinBox" name="spinBoxHeight">
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="minimum">
           <number>100</number>
          </property>
          <property name="maximum">
           <number>10000</number>
          </property>
          <property name="value">
           <number>1080</number>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="spinBoxWidth">
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="minimum">
           <number>100</number>
          </property>
          <property name="maximum">
           <number>10000</number>
          </property>
          <property name="value">
           <number>1920</number>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QCheckBox" name="checkBoxResize">
          <property name="text">
           <string>Resize</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="label_3">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>x</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="5">
         <widget class="QCheckBox" name="checkBoxHdrBlending">
          <property name="toolTip">
           <string>Requires HDR clips!</string>
          </property>
          <property name="text">
           <string>HDR blending</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2" colspan="3">
         <widget class="QDoubleSpinBox" name="doubleSpinBoxFps">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="suffix">
           <string> fps</string>
          </property>
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="minimum">
           <double>1.000000000000000</double>
          </property>
          <property name="maximum">
           <double>60.000000000000000</double>
          </property>
          <property name="value">
           <double>25.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QToolButton" name="toolButtonLockHeight">
          <property name="toolTip">
           <string>Lock apect ratio - calculate height automatically</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="ressources.qrc">
            <normaloff>:/RetinaIMG/RetinaIMG/Actions-document-decrypt-icon.png</normaloff>:/RetinaIMG/RetinaIMG/Actions-document-decrypt-icon.png</iconset>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QCheckBox" name="checkBoxFpsOverride">
          <property name="text">
           <string>Framerate override</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_Smoothing">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Smooth aliasing</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBoxSmoothing">
          <property name="toolTip">
           <string>Use this option to smooth aliasing / moiree artifacts.
This option is very slow and runs when export progressbar is at 100%!
It won't bring good results in shots with much movement and much detail.</string>
          </property>
          <item>
           <property name="text">
            <string>Off</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1 pass</string>
           </property>
           <property name="icon">
            <iconset resource="ressources.qrc">
             <normaloff>:/RetinaIMG/RetinaIMG/Status-dialog-warning-icon.png</normaloff>:/RetinaIMG/RetinaIMG/Status-dialog-warning-icon.png</iconset>
           </property>
          </item>
          <item>
           <property name="text">
            <string>3 pass</string>
           </property>
           <property name="icon">
            <iconset resource="ressources.qrc">
             <normaloff>:/RetinaIMG/RetinaIMG/Status-dialog-warning-icon.png</normaloff>:/RetinaIMG/RetinaIMG/Status-dialog-warning-icon.png</iconset>
           </property>
          </item>
          <item>
           <property name="text">
            <string>3 pass + unsharp mask</string>
           </property>
           <property name="icon">
            <iconset resource="ressources.qrc">
             <normaloff>:/RetinaIMG/RetinaIMG/Status-dialog-warning-icon.png</normaloff>:/RetinaIMG/RetinaIMG/Status-dialog-warning-icon.png</iconset>
           </property>
          </item>
          <item>
           <property name="text">
            <string>last resort</string>
           </property>
           <property name="icon">
            <iconset resource="ressources.qrc">
             <normaloff>:/RetinaIMG/RetinaIMG/Status-dialog-warning-icon.png</normaloff>:/RetinaIMG/RetinaIMG/Status-dialog-warning-icon.png</iconset>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QCheckBox" name="checkBoxExportAudio">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Export audio</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="groupBoxScripting">
     <property name="title">
      <string>Scripting</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QLabel" name="label_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Post Export</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="comboBoxPostExportScript">
          <item>
           <property name="text">
            <string>None</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>212</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonClose">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>comboBoxCodec</tabstop>
  <tabstop>checkBoxExportAudio</tabstop>
  <tabstop>pushButtonClose</tabstop>
 </tabstops>
 <resources>
  <include location="ressources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>checkBoxFpsOverride</sender>
   <signal>toggled(bool)</signal>
   <receiver>doubleSpinBoxFps</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>99</x>
     <y>73</y>
    </hint>
    <hint type="destinationlabel">
     <x>233</x>
     <y>73</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
